File: Cerebro/Models/PrintingEntity.cs
using Microsoft.Azure.Cosmos.Table;

namespace Cerebro.Models
{
    public class PrintingEntity : TableEntity
    {
        public const string TABLE_NAME = "CerebroPrintings";

        public PrintingEntity() { }

        public string ArtificialId { get; set; }

        public string PackName { get; set; }

        public string PackNumber { get; set; }

        public string SetNumber { get; set; }

        public PackEntity Pack { get; set; }
    }
}

File: models/printingEntity.js
class PrintingEntity
{
    constructor (document) {
        this.ArtificialId = document.ArtificialId;
        this.PackId = document.PackId;
        this.PackNumber = document.PackNumber;
        this.SetId = document.SetId;
        this.SetNumber = document.SetNumber;
        this.UniqueArt = document.UniqueArt;
    }
};

module.exports = { PrintingEntity };